cpp/epidemiology/secir/contact_matrix.h

```diff
@@ -300,14 +300,12 @@ class DampingMatrixExpressionGroup
         assert(il.size() > 0);
     }
 
-private:    
```

Comment:
I don't know if we should remove these scopes here. Maybe the issue was that there were functions in public, private, public?

@dabele Could you check that everyting still is as intended?

Reply:
I don't know why the constructor was declared private. I removed the declaration because for MSVC one of the derived classes could not access the private constructor of its base. I assume that in gcc the public initializer list ctor is used, while in MSVC this implicit conversion does not work.